pyaedt/modules/report_templates.py

```diff
@@ -205,10 +205,44 @@ class Trace(object):
     def __init__(self, report_setup, trace_name):
         self._oreport_setup = report_setup
         self.trace_name = trace_name
+        self._trace_name_real = None
         self.LINESTYLE = LineStyle()
         self.TRACETYPE = TraceType()
         self.SYMBOLSTYLE = SymbolStyle()
 
+    @property
+    def trace_name_real(self):
+        report_name = self.trace_name.split(":")[0]
+        traces_in_report = self._oreport_setup.GetReportTraceNames(report_name)
+        for trace in traces_in_report:
+            if trace + ":" in self.trace_name:
```

Comment:
Just a suggestion, but do we really need to check if `:` if part of the `trace_name`?
```suggestion
            if trace in self.trace_name:
```

Reply:
yes because we can have trace names as dB(S(1:1,1:1)) and dB(S(1:1,1:1))_1 and trace_name is composed as:
plot_name:trace_name:variations_list

Reply:
I agree. Anyway I think we should choose a name that is more meaningful. It's not clear to me that real means...

Reply:
Thanks @gmalinve , it makes sense.